sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/TungstenAggregate.scala

```diff
@@ -69,6 +72,9 @@ case class TungstenAggregate(
   protected override def doExecute(): RDD[InternalRow] = attachTree(this, "execute") {
     val numInputRows = longMetric("numInputRows")
     val numOutputRows = longMetric("numOutputRows")
+    val totalPeakMemory = longMetric("totalPeakMemory")
```

Comment:
Hey can you give a more precise definition here of what this means? I think the word "Peak" is throwing me off and maybe we could delete it, if you say "memory" I will assume you mean the maximum amount of memory a task is using over its lifetime. I think on this one it might be best to just discuss it briefly in person.